lib/twilio-ruby/twiml/twiml.rb

```diff
@@ -27,7 +27,10 @@ def initialize(indent: false, **keyword_args)
 
         def self.to_lower_camel_case(symbol)
           # Symbols don't have the .split method, so convert to string first
-          result = symbol.to_s.split('_').map(&:capitalize).join
+          result = symbol.to_s
+          if result.include? '_'
+            result = result.split('_').map(&:capitalize).join
+          end
```

Comment:
This if block needs to include the `result[0].downcase + result[1..result.length]` line from below, as `result = result[0].downcase + result[1..result.length]`

Reply:
shouldnt it just execute the last line `result[0].downcase + result[1..result.length]` after it goes through the if block? Not sure I understand what you mean however I'm not very familiar with ruby.

Reply:
Oh, yes you're right, that will still work fine... I was thinking of an array of string parts with the [0] and [1..result.length] being for that array.  But by that point it's a string again so [0] is just the first character...
Looks good.